contributing/documentation/contributing_to_the_documentation.rst

```diff
@@ -167,13 +167,30 @@ a meaningful name and include them in your page with:
 
    .. image:: img/image_name.png
```

Comment:
```suggestion
   .. image:: img/image_name.webp
```

If you wouldn't mind, since you're already editing the page changing this would be helpful.